sdk/src/main/java/com/mapbox/maps/Style.kt

```diff
@@ -91,6 +91,10 @@ class Style internal constructor(
    *
    * @param transitionOptions Map style transition options.
    */
+  @Deprecated(
+    "Transition options will not be applied properly, please use another function.",
+    replaceWith = ReplaceWith("mapboxMap.setStyleTransition")
```

Comment:
Does it replaces an expression correctly though?

Reply:
`mapboxMap.setStyleTransition(transitionOptions)` should work better

Reply:
yeah, my miss, thanks for catching, will take another look so that it gets replaced correctly